Added setAccel/GyroSensitivity

Smartheads opened the pull request:
**New methods:**

- writeBits
- setGyroSensitivity
- setAccelSensitivity

**New fields:**

- private gyroSensitivity
- private accelSensitivity

**Other changes:**

- Edited begin()
- Edited update()

tockn commented on src/MPU6050_tockn.cpp:
```diff
@@ -98,48 +122,195 @@ void MPU6050::calcGyroOffsets(bool console){
+
+double MPU6050::getAccelAngleX(double accelX, double accelY, double accelZ)
+{
+	return atan2(accelY, accelZ + abs(accelX)) * 180 / PI;
```
Why do `getAccelAngle` methods use arguments?  
I think `getAccelAngle` use no arguments and only return `angleAcc` variable is more good.

tockn replied:
`getAccelAngle` ⇨ `getAccAngle`
Please don't change method name.  
It will lost backward compatibility.

tockn commented on src/MPU6050_tockn.cpp:
```diff
@@ -98,48 +122,195 @@ void MPU6050::calcGyroOffsets(bool console){
+	double accZ = getAccelY();
 
-	temp = (rawTemp + 12412.0) / 340.0;
+	double angleAccX = getAccelAngleX(accX, accY, accZ);
```
Calculate `angleAcc` in this line.  
Please read review at line 305.

tockn commented on src/MPU6050_tockn.cpp:
```diff
@@ -16,32 +9,71 @@ MPU6050::MPU6050(TwoWire &w, float aC, float gC){
-  angleX = this->getAccAngleX();
-  angleY = this->getAccAngleY();
-  preInterval = millis();
+	
```
Please fix white spaces.  
line 13, 17, 49, 58, 64, 160, 178, 182, 186, 190, 194, 213, 217, 221, 225, 229

tockn commented on src/MPU6050_tockn.h:
```diff
@@ -14,78 +14,96 @@
-
-	MPU6050(TwoWire &w);
-	MPU6050(TwoWire &w, float aC, float gC);
+	
```
Please fix white spaces.  
line 35, 41, 87, 98, 102

tockn commented:
Thank you for your awesome commits!  
I would like to merge your commits.  
Please fix some requests!

Smartheads commented:
Ok so I came up with a new pull request. (13.07.2018)
* I refactored the method name according to your naming conventions & to increase backwards compatibility.
* I removed the getAccXYZAngle methods and did all the calculations in the update method.
* I finished the interruptUpdate method and called it from the update method reduce code duplication.
* // Bug fixes (changed int16_t to double in some places)

If you really insist I can remove the whitespaces.
Let me know what you think.
